terraform/050-pw-manager/README.md

```diff
@@ -125,7 +125,7 @@ module "pwmanager" {
   password_rule_minscore              = "${var.password_rule_minscore}"
   recaptcha_key                       = "${var.recaptcha_key}"
   recaptcha_secret                    = "${var.recaptcha_secret}"
-  source                              = "github.com/silinternational/idp-in-a-box//terraform/050-pw-manager?ref=develop-4.0"
+  source                              = "github.com/silinternational/idp-in-a-box//terraform/050-pw-manager?ref=3.1.0"
```

Comment:
I don't think we reference versions in any other readme examples, maybe we should remove from here to be consistent.